### PR TITLE
add Sensio security checker

### PR DESCRIPTION
**Pull request by Tjoosten**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -26,10 +27,12 @@
 	},
 	"scripts": {
 		"post-install-cmd": [
+      "php vendor/bin/security-checker security:check --format=simple",
```
Comment (overtrue) [inline composer.json]: Indentation

File: composer.json [hunk 2]
```diff
@@ -26,10 +27,12 @@
...
 			"php artisan clear-compiled",
 			"php artisan optimize"
 		],
 		"post-update-cmd": [
+      "php vendor/bin/security-checker security:check --format=simple",
```
Comment (overtrue) [inline composer.json]: Indentation

**Comment by barryvdh** [comment]
We could also require it in the framework and add it to the commands, as it's just a Symfony command. And then run it here as `php artisan security:check`. (related: https://github.com/laravel/framework/issues/440)

**Comment by franzliedke** [comment]
What about this one? https://github.com/Roave/SecurityAdvisories

**Comment by GrahamCampbell** [comment]
> https://github.com/Roave/SecurityAdvisories

Would make more sense to me.

**Pull request closed without merging**